exonum/src/api/mod.rs

```diff
@@ -198,34 +198,16 @@ pub trait Api {
         Ok(value)
     }
 
-    /// Deserializes an optional parameter from request body or get-parameters
-    fn optional_param<T>(&self, request: &mut Request, name: &str) -> Result<Option<T>, ApiError>
```

Comment:
Are you sure `optional_param` and `required_param` should be removed? They were public and (imo) quite useful.

Reply:
I thought that the intention behind this PR was to replace these methods with more convinient tool?